static/app/views/performance/traces/fieldRenderers.tsx

```diff
@@ -39,19 +58,101 @@ export function ProjectRenderer({projectSlug, hideName}: ProjectRendererProps) {
   );
 }
 
+export const TraceBreakdownContainer = styled('div')`
+  position: relative;
+  display: flex;
+  min-width: 150px;
+  height: ${ROW_HEIGHT - 2 * ROW_PADDING}px;
+  background-color: ${p => p.theme.gray100};
+`;
+
+const RectangleTraceBreakdown = styled(RowRectangle)`
+  position: relative;
+  width: 100%;
+`;
+
+export function TraceBreakdownRenderer({trace}: {trace: TraceResult<Field>}) {
+  const theme = useTheme();
+
+  return (
+    <TraceBreakdownContainer data-test-id="relative-ops-breakdown">
+      {trace.breakdowns.map(breakdown => {
+        return (
+          <SpanBreakdownSliceRenderer
+            key={breakdown.start + (breakdown.project ?? t('missing'))}
```

Comment:
Perhaps a little clearer?
```suggestion
            key={breakdown.start + (breakdown.project ?? t('missing instrumentation'))}
```